Delete master-docs-build.yml (#64)
aws-quickstart/quickstart-aws-industrial-machine-connectivity: docs/partner_editable/pre-reqs.adoc

=== Review the dataflow options

When you deploy the {partner-product-short-name} Quick Start, you have the option to enable data collection in one of two ways. Each option ingests data differently from the edge environment into the AWS Cloud. For illustrations of these dataflows in the virtual and physical deployments, see (<<architecture-virtual>>) and (<<architecture-physical>>). 

* *Dataflow option 1 (OPC UA to AWS IoT SiteWise).* With this option, you configure the AWS IoT SiteWise connector to connect to the partner edge application OPC UA server. All telemetry data flows directly into AWS IoT SiteWise.

> * *Dataflow option 2a (MQTT to AWS IoT Core).* With this option, the partner edge-software server connects to AWS IoT Core from MQTT. All telemetry data is pushed from the edge-software server to AWS IoT Core. From there, the data is pushed to Amazon S3 for processing. Data is not forwarded to AWS IoT SiteWise. Custom development is required to populate data in AWS IoT SiteWise.

Reviewer: @sshvans Again, please change "2a" to "2" throughout the text and images. (I'm not marking them all.)

For physical deployments, you configure both the deployment type and the dataflow option during the post-deployment steps by using one of the following bootup scripts:
 
* BootupScriptGreenfieldOption1 (greenfield deployment, dataflow option 1)

> * BootupScriptGreenfieldOption2a (greenfield deployment, dataflow option 2a)

Reviewer: @sshvans Some renaming in the code may be needed as well to change "2a" to "2." Leaving "2a" as is would be convenient, but it's confusing in the absence of a "2b."

* BootupScriptBrownfieldAllOptions (brownfield deployment, any dataflow option)

=== Review the AMC drivers

> When you deploy the {partner-product-short-name} Quick Start, you choose an AMC driver. This driver determines which edge application the {partner-product-short-name} Quick Start connects to AWS IoT SiteWise. For the AWS CloudFormation stack `AMCDriver` parameter, you can choose any of four drivers: IgnitionCirrusLink, IgnitionFileExport, or KepwareFileExport.

Reviewer: @sshvans Since we say "four drivers" now, please add a fourth to this list.

Author: Fixed

* *IgnitionFileExport*. This AMC driver uses the manual tag-definition file-export mechanism within Ignition. With this AMC driver, you manually export the tag-definition file and upload it to an S3 bucket to initiate the AMC process during the post-deployment steps. For the AWS CloudFormation stack's `AMCDriver` parameter, you choose `IgnitionFileExport`.

* *KepwareFileExport*. This AMC driver uses the manual tag-definition file-export mechanism within Kepware. With this AMC driver, you manually export the tag-definition file and upload it to an S3 bucket to initiate the AMC process during the post-deployment steps. For the AWS CloudFormation stack's `AMCDriver` parameter, you choose `KepwareFileExport`.

> *ElementUnify*. You can also choose to use Element Unify for Asset Modeling. This will not deploy any drivers for the Asset Model Converter. With Element Unify, you manually export the tag-definition file from either edge application and upload it to an S3 bucket to initiate the automated import of the source system file into Element Unify.

Reviewer: ```suggestion
*ElementUnify*. You can use Element Unify for asset modeling. It does not deploy any drivers for the Asset Model Converter. With Element Unify, you manually export the tag-definition file from either edge application and upload it to an S3 bucket. The source system file is automatically imported into Element Unify.
```

> NOTE: If you choose to include Element Unify in the Quick Start deployment, a valid Element Unify account is required.

Reviewer: ```suggestion
NOTE: To include Element Unify in the Quick Start deployment, you need a valid Element Unify account.
```

.. Open the AWS SSO service in the AWS Management Console.
.. Choose *Users* in the navigation bar.
.. Choose *Add user*.

> .. Provide a username, and fill in the remaining boxes.

Reviewer: ```suggestion
.. Provide a user name, and fill in the remaining boxes.
```

Reviewer: @sshvans Per the AWS Style Guide, the term "user name" is two words. (We treat "username" as one word only if we're referring to a UI label that's one word, in which case we would also bold the word as a UI element.)

.. Choose *Next, Groups*.
.. Choose a group, and then choose *Add user*. The AWS SSO user is now active. This user will be able to access the AWS IoT SiteWise Monitor dashboards.

=== Set up QuickSight

> . Open the QuickSight console and choose *Sign up for QuickSight*.

Reviewer: ```suggestion
. Open the QuickSight console, and choose *Sign up for QuickSight*.
```

Reviewer: @sshvans A comma goes between two independent clauses joined by "and." (It's not mandatory, but it is grammatically appropriate.)
